### PR TITLE
Updated README text

### PR DESCRIPTION
**Pull request by jpayne00**
Changes were applied to the README at the request of LADOT Executive Staff.
The intent is to make the language more comprehensible to the general public.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,43 @@
...
 
-**MDS** is currently comprised of two distinct components:
+**MDS Components** 
 
-* The [`provider`][provider] API is to be implemented by *mobility as a service* providers, for data exchange and operational information that a municipality will query. `provider` presents the historical view of operations.
+* The [`provider`][provider] Application Program Interface (API) was created in May 2018 to be implemented by mobility providers. When a municipality queries information from a mobility provider, the Provider API automatically generates a historical view of operations in a standard format.
 
-* The [`agency`][agency] API is to be implemented by *municipalities* and other regulatory agencies, for providers to query and integrate with during operations. `agency` provides tools to inform and permit future operations.
+* The [`agency`][agency] API was created in April 2019 to be implemented by regulatory agencies. Providers query the Agency API when an event occurs, like a trip starting or ending.
```
Comment (karcass) [inline README.md]: Hm, I wouldn't say it was created in April of 2019.  It was conceived in the same time frame as Provider.  It did go through some revisions, and it wasn't implemented by LADOT until February of 2019.  April 2019 is when the reference implementation went into production.

**Comment by hunterowens** [comment]
I believe this is related to #287

**Comment by jannasmith** [comment]
Some of this information may be better suited to the ladot.io page about MDS, such as "How MDS is used in Los Angeles" and the LADOT-specific data sharing requirements. Are people pointing to ladot.io as a "general public"-facing landing page, or still this github link? The github readme should be focused on an audience of operators, cities' technical staff, and other potential contributors.

We should leave in the information about what kind of API spec MDS is, and reference to GBFS - these are more beneficial for contributors who want to suggest edits and updates to the spec and cities/operators who want to look at the code itself. This audience also probably doesn't need a lot of detail about LADOT's dockless permitting, etc

File: README.md (hunk 1 above)
Comment (hunterowens) [reply README.md]: @karcass @jpayne00 how does 

`The [`agency`][agency] API is implemented by the regulatory agency, ie,  LADOT or the County of Los Angeles. Providers query the API during operations to get information from the regulatory agency.`

**Comment by hunterowens** [comment]
closing in favor of #332

**Pull request closed without merging**